### DISCUSSION TITLE
How to set the parameters when training with only energy?

### DISCUSSION THREAD

**Discussion by JunbangLiu** [deepmodeling/deepmd-kit#1611] [Q&A]
I set `pref_f` to 0 and remained other parameters unchanged, but the training result is unsatisfying.  `rmse_e` is about 1e-6 per atom after 1 000 000 steps training with both energy and force, while it is about 1e-4 per atom training with only energy. I need help to improve the result if possible. Here's my `input.json`:
```
{
    "_comment": " model parameters",
    "model": {
	"type_map":	["Mo", "S"],
	"descriptor" :{
	    "type":		"se_e2_a",
	    "sel":		"auto",
	    "rcut_smth":	5.20,
	    "rcut":		6.00,
	    "neuron":		[25, 50, 100],
	    "resnet_dt":	false,
	    "axis_neuron":	16,
	    "seed":		1,
	    "_comment":		" that's all"
	},
	"fitting_net" : {
	    "neuron":		[240, 240, 240],
	    "resnet_dt":	true,
	    "seed":		1,
	    "_comment":		" that's all"
	},
	"_comment":	" that's all"
    },

    "learning_rate" :{
	"type":		"exp",
	"decay_steps":	5000,
	"start_lr":	0.001,	
	"stop_lr":	3.51e-8,
	"_comment":	"that's all"
    },

    "loss" :{
	"type":		"ener",
	"start_pref_e":	0.02,
	"limit_pref_e":	1,
	"start_pref_f":	0,
	"limit_pref_f":	0,
	"start_pref_v":	0,
	"limit_pref_v":	0,
	"_comment":	" that's all"
    },

    "training" : {
	"training_data": {
	    "systems":		["../deepmd_data_noex"],
	    "batch_size":	"auto",
	    "_comment":		"that's all"
	},
	"validation_data":{
	    "systems":		["../deepmd_data_noex_v"],
	    "batch_size":	"auto",
	    "_comment":		"that's all"
	},
	"numb_steps":	1000000,
	"seed":		10,
	"disp_file":	"lcurve.out",
	"disp_freq":	1,
	"save_freq":	1000,
	"_comment":	"that's all"
    },    

    "_comment":		"that's all"
}
```
In addtion, I noticed that `rmse_e` drops quickly to 1e-4 in first 200 steps and doesn't decrease anymore when training with only energy, maybe I should slow it down? But I don't know what to do.

**Comment by McAlexDudu** [comment]
why you do that? I think it's necessary to specify the pref_f, only energy rmse to evaluate the loss is not enough ,and may due to hard to decrease the total loss.
